pkg/container/container.go

```diff
@@ -173,7 +173,39 @@ func (c *containerModule) Run(ns string, data pkg.Container) (id pkg.ContainerID
 		return id, err
 	}
 
-	task, err := container.NewTask(ctx, cio.LogFile(path.Join(logs, fmt.Sprintf("%s.log", container.ID()))))
+	// inifialize multi-loggers
+	loggers, err := logger.NewContainerLoggers(ctx)
+	if err != nil {
+		return id, err
+	}
+
+	// hardcode local logfile
+	filepath := path.Join(logs, fmt.Sprintf("%s.log", container.ID()))
+	filelog, err := logger.NewContainerLoggerFile(filepath)
+	if err != nil {
+		return id, err
+	}
+
+	loggers.Add(filelog)
+
+	// set user defined endpoint logging
+	for _, l := range data.Logs {
+		if l.Type == "redis" {
```

Comment:
you probably want to use a switch here, so where there is new type of logger we can just add a case